main.js

```diff
@@ -0,0 +1,48 @@
+document.getElementById("searchButton").addEventListener("click", function () {
+  var locationInput = document.getElementById("locationInput").value;
+  fetchWeatherData(locationInput);
+});
+
+function fetchWeatherData(location) {
+  var apiKey = "235b291d30b715c64c5632322d291bc1";
+  var apiUrl = `http://api.openweathermap.org/data/2.5/forecast?q=${location}&appid=${apiKey}&units=imperial`;
+
+  fetch(apiUrl)
+    .then((response) => response.json())
+    .then((data) => {
+      displayWeatherData(data);
+    });
+}
+
+function displayWeatherData(data) {
```

Comment:
would recommend changing data to something less generic - make as descriptive as possible